Sources/SwiftSDKGenerator/SwiftSDKRecipes/LinuxRecipe.swift

```diff
@@ -0,0 +1,58 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2022-2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import AsyncHTTPClient
+import GeneratorEngine
+
+public struct LinuxRecipe: SwiftSDKRecipe {
+  public init() {}
+
+  public func cookSwiftSDK(generator: SwiftSDKGenerator, engine: Engine, httpClient client: HTTPClient) async throws {
```

Comment:
```suggestion
  public func prepareSwiftSDK(generator: SwiftSDKGenerator, engine: Engine, httpClient client: HTTPClient) async throws {
```

Reply:
I feel `prepare` implies that the operation is not the main part but a minor part and subsequent operations are the main. How about `make`?

Reply:
Yes, that makes sense, and I know there's a lot of precedent for `make` in swift APIs and also with build systems, I like `make` 👍